### PR TITLE
Update excludeReferrers language

### PR DESCRIPTION
**Pull request by jnthns**
# Amplitude Developer Docs PR

## Description

[Thread ](https://amplitude.slack.com/archives/C02E1H87E2C/p1712079024299769)

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sdks/browser-2/index.md [hunk 1]
```diff
@@ -460,4 +460,4 @@ Amplitude doesn't track marketing attribution under any of the following conditi
 | The referrer originates from the same domain. | The landing page is `a.test.com`, with the referrer set to `b.test.com`. |
 | A specific referrer domain is explicitly excluded.| When setting `config.defaultTracking.attribution.excludeReferrers` = `[a.test.com]`, and the referrer domain is `a.test.com` for the current page. |
 | The subdomain is specified or matches the regular expression in `config.defaultTracking.attribution.excludeReferrers`.| Configuration of excludeReferrers involves specific string arrays or a regular expression. |
-| The user engages in direct traffic within the same session.| During a session, a user clicks on a link without any campaign attribution parameters, including the absence of UTM and click id parameters from an email. |
\ No newline at end of file
+| The user engages in direct traffic within the same session.| During a session, a user clicks on a link without any campaign attribution parameters, including the absence of UTM and click id parameters from an email. |
```
Comment (Mercy811) [inline docs/data/sdks/browser-2/index.md:463]: nit: what is changed here?

File: docs/data/sdks/browser-2/index.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ You can also use advanced configuration for better control of how marketing attr
 ???config "Marketing attribution options"
     |<div class="big-column">Name</div>|Value|Description|
     |-|-|-|
-    `config.defaultTracking.attribution.excludeReferrers` | Optional. Array of `string` or `RegExp` | Sets rules to determine which referrers are excluded from being tracked as traffic source. Use string values for exact matching and RegExp values for pattern matching against the referring domain. By default the current domain (and all its subdomains) are excluded referrers. |
+    `config.defaultTracking.attribution.excludeReferrers` | Optional. Array of `string` or `RegExp` | Sets rules to determine which referrers are excluded from being tracked as traffic source. Use string values for exact matching and RegExp values for pattern matching against the referring domain. When this option isn't in use, the current domain (and its subdomains) are excluded referrers. If explicitly adding an external referrer to exclude, you must also add the current domain (and its subdomains) as additional referrers to exclude. |
```
Comment (Mercy811) [inline docs/data/sdks/browser-2/index.md:218]: nit
```suggestion
    `config.defaultTracking.attribution.excludeReferrers` | Optional. Array of `string` or `RegExp` | Sets rules to determine which referrers are excluded from being tracked as traffic source. Use string values for exact matching and RegExp values for pattern matching against the referring domain. When this option isn't set, the current domain (and its subdomains) are excluded referrers. If explicitly adding an external referrer to exclude, you must also add the current domain (and its subdomains) as additional referrers to exclude. |
```

**Review by Mercy811 — APPROVED** [review]
Thanks @jnthns for updating this. Overall looks good to me. See nits.

File: docs/data/sdks/browser-2/index.md (hunk 1 above)
Comment (jnthns) [reply docs/data/sdks/browser-2/index.md:463]: hmm didn't change anything here - think this is a formatting change?

Comment (jnthns) [reply docs/data/sdks/browser-2/index.md:463]: @markzegarelli could you help us check if this is going to impact the table format?

**Review by markzegarelli — APPROVED** [review]

File: docs/data/sdks/browser-2/index.md (hunk 1 above)
Comment (markzegarelli) [reply docs/data/sdks/browser-2/index.md:463]: @jnthns No impact to the table format. looks good to me.

**Pull request merged**